.github/actions/bench/action.yml

```diff
@@ -29,11 +29,11 @@ inputs:
   gh_token:
     description: GitHub access token
     required: true
-  use-nix:
-    description: Whether to run in the default Nix environment
-    default: "true"
+  nix-shell:
+    description: Run in the specified Nix environment if exists
+    default: "ci"
```

Comment:
It's not possible to unset this, is it? If you pass `""`, the variable will default to `"ci"`, which is not intended.